src/pyJiraCli/error_handler.py

```diff
@@ -34,42 +34,115 @@
+    Ret.RET_ERROR_MISSING_DATATYPE       : "No datatype for login command given. " + \
+                                           "Provide the Datatype via " +\
+                                           "--userinfo, --token, --server or -- default " + \
+                                           "option with the login command",
     Ret.RET_ERROR_CREATING_TICKET_FAILED : "creating the ticket on the jira server failed",
-    Ret.RET_ERROR_INFO_FILE_EXPIRED      : "the stored information has expired"
+    Ret.RET_ERROR_INFO_FILE_EXPIRED      : "the stored information has expired",
+    Ret.RET_ERROR_INVALID_SEARCH         : "search string returned a jira error",
 }
+
+WARN_MSG = {
+    Warnings.WARNING_UNSAVE_CONNECTION      : "No certificate for server " + \
+                                              "authentification found. " + \
+                                              "It's strongly advised, " +\
+                                              "to add a certificate with the login command.",
+    Warnings.WARNING_SERVER_URL_MISSING     : "No Server url found. PLease add a " + \
+                                              "custom url for your jira server.",
+    Warnings.WARNING_CSV_OPTION_WRONG       : "File ending from provided file and " + \
+                                              "csv option dont match. " + \
+                                              "File format provided by file or path was used.",
+    Warnings.WARNING_UNKNOWN_FILE_EXTENSION : "The provided file has an unknown file format. " +\
+                                              "A new file with the same " +\
+                                              "name and a file format according to the " +\
+                                              "-csv option will be created."
+}
+
+INFO_TAB = "      "
 ################################################################################
 # Classes
 ################################################################################
+class Error:
+    """ The error handler class.
+    """
+
+    _print_verbose = False
+
+    def __init__(self):
+        pass
+
+    @classmethod
+    def set_verbose(cls):
+        """Set verbose mode for all instances of the class."""
+        cls._print_verbose = True
+
+    def print(self, err_type:ErrorType, error:Ret=Ret.RET_OK) -> None:
+        """ Print the exit error.
+    
+        Args:
+            type (ErrorType)    The type of the msg (Error, Warning or Info).
+            error (Ret):        The return code for which an error shall be printed.
+        """
+        if err_type is ErrorType.WARNING and \
+           self._print_verbose:
+            print(COLOR[err_type] + TYPE[err_type] + ": " + Style.RESET_ALL + WARN_MSG[error])
+
+        elif err_type is ErrorType.ERROR:
+            print(COLOR[err_type] + TYPE[err_type] + ": " + Style.RESET_ALL + RETURN_MSG[error])
+
+    def print_info(self, *args:str) -> None:
```

Comment:
Don't mix error information and standard info in a error class. Maybe the error class is more like a logger, than an error handler.